src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,52 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private double volume;
+    private int year;
+    private int weight;
+    private String madeBy;
+    private boolean turbo;
```

Comment:
```suggestion
    private boolean isTurbo;
```
the getter should be named `isTurbo`, setter - `setTurbo`

Reply:
done